sdk/communication/communication-call-automation/review/communication-call-automation.api.md

```diff
@@ -102,6 +102,9 @@ interface AudioData_2 {
 }
 export { AudioData_2 as AudioData }
 
+// @public
+export type AudioFormat = string;
```

Comment:
Let's make sure to also export the `KnownAudioFormat`, same for the other extensible enums.